Add black beans and rice recipe

ZooeyMiller opened the pull request:
Added the black beans and rice recipe cooked this week
fix #12

ZooeyMiller committed:
adds basic black bean recipe
#12

ZooeyMiller committed:
follow @m4v15 styleguide
#12

ZooeyMiller committed:
adds link to recipe in readme.md
#12

ZooeyMiller committed:
fixes markdowns
#12

m4v15 commented on README.md:
```diff
@@ -4,6 +4,7 @@ You know what this is for.
 ## Breakfast
 
 ## Mains
+[Black beans w/ Lime & Coriander rice (and sides)](black-beans.md)
```
Maybe put it in a recipes/mains/ folder, like mine?

m4v15 commented on black-beans.md:
```diff
@@ -0,0 +1,89 @@
+# Black beans with lime and coriander rice
```
This seems more than just black beans and rice recipe?

ZooeyMiller replied:
fix'd

m4v15 reviewed (changes requested):
Add to folder, maybe change the title to include the peppers and onion stuff?

ZooeyMiller committed:
fixes changes requested by @m4v15
https://github.com/foundersandcoders/nazareth-recipes/pull/14#discussion_r130076029

ZooeyMiller commented:
@m4v15 please re-review given the changes.

m4v15 commented:
@ZooeyMiller I will when there are no more conflicts.

ZooeyMiller committed:
Merge branch 'master' into black-beans

ZooeyMiller commented:
@m4v15 solved merge conflicts. Please review.
